src/js/shared/application-form.js

```diff
@@ -73,47 +72,47 @@ exports.createApplicationForm = function createApplicationForm(validateFile = tr
     }),
     countryTravellingFrom: fields.string({
       widget: widgets.select(),
-      label: 'Where will you be travelling from before you reach Cambridge?',
-      note: 'This may or may not be where you live at the moment.',
+      label: 'Where will you be coming from?',
+      note: 'This does not have to be your current country of residence.',
       required: requiredField,
       choices: createCountryChoices(),
       cssClasses,
     }),
     development: fields.array({
-      label: 'Where do you see yourself fitting into the development process?',
+      label: 'What role or roles in a team would you be interested in?',
       note: 'Tick all that apply.',
       widget: multiCheckboxWidget(),
       required: requiredField,
       choices: {
         development: 'Development',
         design: 'Design',
         product_management: 'Product Management',
-        unknown: 'I\'m not sure!',
+        unknown: 'I’m not sure',
       },
       validators: [
         (form, field, callback) => {
           if ((field.data.includes('unknown')) && (field.data.length > 1)) {
-            callback('You can\'t have an answer and not be sure!');
+            callback('You can’t have an answer and not be sure!');
           } else {
             callback();
           }
         },
       ],
       cssClasses,
     }),
-    learn: textareaField('What do you want to learn from this event?', 500, {
+    learn: textareaField('How will the event benefit you?', 500, {
       required: requiredField,
     }),
-    interests: textareaField('What interests you?', 500, {
-      note: 'This can be anything at all — it doesn\'t have to be technology-related!',
+    interests: textareaField('What are you interested in?', 500, {
+      note: 'Mention anything you want—it doesn’t have to be technology-related!',
```

Comment:
Spaces before and after em dash

Reply:
http://www.thepunctuationguide.com/em-dash.html

Reply:
Personally, I've seen spaces around the em dash far more often in the style that the page describes:

> Most newspapers — and all that follow AP style — insert a space before and after the em dash.